models/stg_reddit_ads.yml

```diff
@@ -81,8 +81,8 @@ models:
         tests:
           - unique
           - not_null
-      - name: _fivetran_synced
-        description: '{{ doc("_fivetran_synced") }}'
+      - name: _fivetran_synced 
+        description: '{{ doc("_fivetran_synced") }}' 
```

Comment:
We don't actually bring this field through to the staging model. We should be able to remove from the docs.

Reply:
removed